test/parallel/test-repl-autolibs.js

```diff
@@ -42,7 +42,7 @@ function test1() {
                          `${util.inspect(require('fs'), null, 2, false)}\n`);
       // Globally added lib matches required lib
       assert.strictEqual(global.fs, require('fs'));
-      test2();
+      process.nextTick(test2);
```

Comment:
Can you clarify if this is a major change?

Reply:
For consumers that have defined their application in the exact same was as this test, then yes this can be considered a major change. Arguably, only the last assert can be the one using `process.nextTick` and not `test2` call. This is that is happening behind (wrapped each `putIn.run` with 2 console.logs, `run` and `end`):

```
run fs
REPL 130410: line "fs"
REPL 130410: eval "fs\n"
REPL 130410: finish null {...} // fs output, redacted
run url
REPL 130410: queued line "url"
end url
REPL 130410: line "url"
REPL 130410: eval "url\n"
REPL 130410: finish null {}
end fs
```

So, for enqueued lines/cmds, the control will be passed back immediately.

Reply:
Yeah agreed the line on this one is less clear. But if the explicit callback change is already major then that sets the bar here as well anyway.